aireview/README.md

```diff
@@ -1,2 +1,54 @@
 # AIReview
-Pull request review agent
\ No newline at end of file
+Pull request review agent
```

Comment:
**documentation**: Consider adding environment setup instructions (e.g., environment variables, configuration files) before the deployment section.